src/components/CopyText/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import { Colors } from '../theme'
+import { IconCopyAccountAddress } from '../Icons'
+import copyToClipboard from '../../utils/copyToClipboard'
+
+const CopyTextEl = styled.span`
+  position: relative;
+  cursor: pointer;
+
+  svg {
+    transition: transform 0.1s ease-out;
+  }
+
+  .copy-text {
+    position: absolute;
+    top: calc(50% - 0.2em);
+    left: calc(100% + 0.25em);
+    opacity: 0;
+    color: ${props => props.color};
+    transform: translateY(-50%);
+    transition: opacity 0.2s ease-in;
+    pointer-events: none;
```

Comment:
I _think_ this might be changing the behavior of the old CopyText component, and I personally prefer the old behavior.

The old behavior - hovering over the CopyIcon _or_ the <span> tag would cause the "copy"/"copied" text to stay visible.

The new behavior - hover over the CopyIcon causes the "copy"/"copied" text to stay visible. When i scroll right to click "copy", it disappears, which is kinda annoying.

Removing `pointer-events: none` seems to restore the old behavior, but i held off on making this change because I wasn't sure if keeping this css property was important for some other reason? Or if there was a better approach to restoring the old behavior

Reply:
Ah I see, yeah with the old behaviour the `Copy` / `Copied` text would basically take up space and be clickable while it had `opacity: 0`. This could cause weird behaviour if you'd want to right-align the copy icon in an element or have other buttons right next to it. I'll have a quick think if we can kind of have the best of both worlds and keep it visible after it pops up.

Or do you also prefer to have the inivisble text as a hit-area before it shows?

Reply:
I've updated the behaviour. Now the `Copy` / `Copied` text does stay visible when hovering over it, but you do need to hover over the icon to show it in the first place.